poc_iot_verifier/src/loader.rs

```diff
@@ -18,22 +19,24 @@ use helium_proto::{
     services::poc_lora::{LoraBeaconIngestReportV1, LoraWitnessIngestReportV1},
     EntropyReportV1, Message,
 };
-use node_follower::{follower_service::FollowerService, gateway_resp::GatewayInfoResolver};
 use sha2::{Digest, Sha256};
 use sqlx::PgPool;
 use std::time::Duration;
 use tokio::time;
 
 /// cadence for how often to look for new beacon and witness reports from s3 bucket
-const REPORTS_POLL_TIME: time::Duration = time::Duration::from_secs(60);
+const REPORTS_POLL_TIME: time::Duration = time::Duration::from_secs(60 * 5 + 10);
```

Comment:
```suggestion
const REPORTS_POLL_TIME: time::Duration = time::Duration::from_secs((60 * 5) + 10);
```
not sure what the math should be here? should it be 310 or 900?